ronin-dev.sh

```diff
@@ -252,9 +252,9 @@ function auto_install_git()
 #
 function auto_install_ruby()
 {
-	# check if ruby-3.x is already installed
-	if [[ ! "$ruby_version" == "3."* ]]; then
-		log "Installing ruby 3.x ..."
+	# check if ruby >= 3.1 is already installed
+	if [[ -z "$ruby_version" ]] || [[ "$ruby_version" == "2."* ]] || [[ "$ruby_version" == "3.0."* ]]; then
```

Comment:
The same logic also needs to be copied to `ronin-install.sh` since it also has an `auto_install-ruby()` function.